tests/parser/functions/test_extract32.py

```diff
@@ -2,16 +2,16 @@ def test_extract32_extraction(assert_tx_failed, get_contract_with_gas_estimation
     extract32_code = """
 y: Bytes[100]
 @external
-def extrakt32(inp: Bytes[100], index: int128) -> bytes32:
```

Comment:
Why was this type changed?

Reply:
Negative index is not allowed for the index argument `extract32` even though the index type was `int128`. As I understand, `int128` was the only integer type when this was implemented. Since Vyper has unsigned integer types now, index type has been updated to be of `UnsignedIntegerAbstractType` instead and would be more reflective as well.

Reply:
`uint256 ` was added several years ago, so is kind of strange it was like that